Yoast/wordpress-seo
# Added seo-for-beginners banner to backend

## Summary

This PR can be summarized in the following changelog entry:

* Adds banner for 'SEO for beginners course' to sidebar.

## Test instructions

This PR can be tested by following these steps:

* Go to the Yoast SEO dashboard. At the right side of the screen you should see the Yoast advertisements. Refresh a couple of times until you see the "SEO for beginners" banner under "Improve your SEO skills".

## Quality assurance


Fixes #9597
